### ISSUE TITLE
Black Screen

### ISSUE THREAD

**Issue by TheRealLinzer** [libbymiller/pi-frame#1]
Hey,
I followed your tutorial but everytime I execute "bash /home/pi/frame/frame.sh" my screen turns black and that's it.

Is it nescessary to fill in the Flickr Access Token on top of the API and secret API key?

Best regards,
Michael

**Comment by libbymiller** [comment]
hi,

It's a long time since I've looked at this code, but yeah, you will need an API key and need to fill them in in ```cacheimages.py``` - otherwise there won't be any images to display, which will indeed make it a black screen.

Libby

**Comment by TheRealLinzer** [comment]
Thank you very much for your quick reply.
I did insert my API key but  the cacheimages.py also has an option to fill in a Flickr Access Token - can you tell me if that is needed as well?

Michael

**Comment by libbymiller** [comment]
No, you don't - I just tried it. You do need ```api_key``` and ```api_secret``` and ```user_id``` though (you can get your ```user_id``` from here - https://www.webpagefx.com/tools/idgettr/ ). 

```python cacheimages.py``` (or ```python cacheimages_zero.py``` if you don't want to use face detection) should prompt you to put a special flickr url into a browser which then gives you a code, looking something like this: ```237-646-941``` which you need to paste back onto the commandline where you were running the ```cacheimages.py``` program. 

At that point it will download a load of pictures from that account into ```public/images```. If this doesn't happen then you'll get a black screen when you run ```sh frame.sh ``` which just runs a little static server with the contents of that directory.
